src/dcos_e2e/cluster.py

```diff
@@ -127,36 +159,34 @@ def wait_for_dcos_ee(
                 healthy in time.
         """
 
+        self._wait_for_dcos_diagnostics()
+
         credentials = {
             'uid': superuser_username,
             'password': superuser_password,
         }
 
         any_master = next(iter(self.masters))
 
-        api_session = DcosApiSession(
+        enterprise_session = EnterpriseApiSession(
             dcos_url='https://{ip}'.format(ip=any_master.ip_address),
             masters=[str(n.ip_address) for n in self.masters],
             slaves=[str(n.ip_address) for n in self.agents],
             public_slaves=[str(n.ip_address) for n in self.public_agents],
             auth_user=DcosUser(credentials=credentials),
         )
 
-        warnings.simplefilter('ignore', InsecureRequestWarning)
-
-        ca_cert = api_session.get(
-            # We wait up to 30 minutes which is extraordinarily large
-            # but should avoid problems with 3 or 5 master counts.
+        response = enterprise_session.get(
```

Comment:
I'd like a comment here and in the OSS equivalent on why `dcos-diagnostics` isn't enough. Ideally linking to JIRA issues.